doc/api/os.md

```diff
@@ -378,9 +378,8 @@ added: v0.3.3
 
 The `os.uptime()` method returns the system uptime in number of seconds.
 
-*Note*: Within Node.js' internals, this number is represented as a `double`.
-However, fractional seconds are not returned and the value can typically be
-treated as an integer.
+*Note*: the return value includes fractions of a second on Windows.
```

Comment:
So I've checked macOS, Ubuntu 16.04, SmartOS 16, FreeBSD 11, and AIX 6.1, and they're all consistent (`os.uptime()` is an integer, `process.uptime()` is not). Given that, I'd say we're okay to specify that it's just windows.

If it changes in the future then the PR to change it should update the docs, making it a `semver-major` change.